From 23f4b4700a9f995df69650bcddbee8290a56fae1 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Fri, 10 Dec 2021 03:19:45 +0000
Subject: [PATCH] Makefile: don't override CPPFLAGS

There's no need to explicitly blank this and it prevents overrides
from the environment.

Signed-off-by: Sam James <sam@gentoo.org>
(cherry picked from commit 9bd435ddbad5a2103d1e562c62f0126fd19f20eb)
---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index ddf0ca552aefe..61ed786168d22 100644
--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,6 @@ CXXFILES?= iptstate.cc
 
 # THIS IS FOR NORMAL COMPILATION
 LIBS?= $(shell $(PKG_CONFIG) --libs ncurses libnetfilter_conntrack)
-CPPFLAGS=
 
 ### YOU SHOULDN'T NEED TO CHANGE ANYTHING BELOW THIS
 
-- 
2.34.1

